ios/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "1430"
+   LastUpgradeVersion = "1510"
```

Comment:
all of the `LastUpgradeVersion` changes should've been fixed in https://github.com/TBD54566975/didpay/commit/b33b6664a6883d8144464e1fed939049560842c7, maybe try merging in the latest main?